### PR TITLE
fixes #7660: fixes first bunch of rubocop warnings

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: extra/register-service.rb [hunk 1]
```diff
@@ -21,9 +21,9 @@
 
   puts "This service must run as a user with permission to execute the netsh dhcp script"
   puts 'The acount can be local or a domain account. If it is a domain account then use the domain\account syntax'
-  user  = ask("Run this service as this user? ") {|user| user.default = default_user}
-  pass1 = ask("Enter the user's password ") {|pass1| pass1.echo = "x"}
-  pass2 = ask("Reenter the password "){|pass2| pass2.echo = "x"}
+  user  = ask("Run this service as this user? ") {|u| u.default = default_user}
```
Comment (daviddavis) [inline extra/register-service.rb:24]: Hmmm... what's this change for?

Comment (dmitri-d) [inline extra/register-service.rb:24]: Variable shadowing cop wasn't happy.

Comment (daviddavis) [inline extra/register-service.rb:24]: oh right. thx!

**Comment by dmitri-d** [comment]
lgtm, but someone else should take a look at the changes too.

**Comment by daviddavis** [comment]
:+1: 

ping @domcleal @eLobato

**Comment by domcleal** [comment]
Thanks both @daviddavis and @witlessbird.  Merged as 25691ff for 1.7.0.

**Pull request closed without merging**